.travis.yml

```diff
@@ -82,6 +82,7 @@ matrix:
       env:
         - SPLIT="4/4"
 before_install:
+  - pip uninstall -y setuptools
```

Comment:
I think this uninstalls setuptools from all of the tests. As far as I can tell, sympy is only installed in the Sage test.

Maybe the best thing would be to set up a new test in the matrix that only tests installing SymPy with and without setuptools.